src/core/atomic.d

```diff
@@ -27,6 +27,13 @@ enum MemoryOrder
      * Corresponds to $(LINK2 https://llvm.org/docs/Atomics.html#monotonic, LLVM AtomicOrdering.Monotonic)
      * and C++11/C11 `memory_order_relaxed`.
      */
+    rlx = 0,
+    /**
+     * Deprecated.
+     * Not sequenced.
+     * Corresponds to $(LINK2 https://llvm.org/docs/Atomics.html#monotonic, LLVM AtomicOrdering.Monotonic)
+     * and C++11/C11 `memory_order_relaxed`.
+     */
     raw = 0,
```

Comment:
Doesn't need to be deprecated, there's little benefit to be gained here.

```suggestion
    /// ditto
    raw = 0,
```

Reply:
We talked about this a bit during beerconf, but didn't reach a clear cut conclusion on how should we tackle this.
The following options were raised.
1. Deprecating the symbol - Resulting in clear code at the cost of deprecation. It will also allow easier reflection over the type.
2. Just adding the new enum member as documented - This results in two different names to the same memory access that can and will be used interchangeably. Nothing will nudge the programmer to use one over the other.
3. Keep `raw` available but undocumented - This will keep old code compiling and nudge users to the new symbol. The cost is that if someone does encounter the old symbol, they'll have to check the source for the meaning.

As a first step, the PR just introduced the new symbol to make sure nothing breaks as a result of this. Deprecating the symbol cause CI failure since it treats warnings as errors.

Reply:
(Should've mentioned that these issues arised at BeerConf. I just wanted to let other people have a word in this discussion as well as https://github.com/dlang/druntime/pull/3485#discussion_r642149544)